randovania/data/json_data/prime1.txt

```diff
@@ -1279,379 +2367,833 @@ Asset id: 1125030300
+  > Event (Elite Quarters Access Barrier Opened)
+      Plasma Beam
 
 > Dock to Elite Quarters; Heals? False
-  * Normal Door to Elite Quarters/Dock to Elite Quarters Access
+  * Plasma Door to Elite Quarters/Dock to Elite Quarters Access
+  > Dock to Metroid Quarantine B
+      After Elite Quarters Access Barrier Opened
+
+> Event (Elite Quarters Access Barrier Opened); Heals? False
+  * Event Elite Quarters Access Barrier Opened
+  > Dock to Metroid Quarantine B
+      Trivial
 
 ----------------
 Quarantine Access A
 Asset id: 1522461728
-> Dock to Central Dynamo; Heals? False
-  * Normal Door to Central Dynamo/Dock to Quarantine Access A
+> Dock to Central Dynamo; Heals? False; Spawn Point
+  * Ice Door to Central Dynamo/Dock to Quarantine Access A
+  > Dock to Metroid Quarantine A
+      Trivial
 
 > Dock to Metroid Quarantine A; Heals? False
-  * Normal Door to Metroid Quarantine A/Dock to Quarantine Access A
+  * Wave Door to Metroid Quarantine A/Dock to Quarantine Access A
+  > Dock to Central Dynamo
+      After Central Dynamo Bendezium Destroyed
 
 ----------------
 Save Station Mines B
 Asset id: 2077614267
 > Dock to Central Dynamo; Heals? False
-  * Normal Door to Central Dynamo/Dock to Save Station Mines B
+  * Ice Door to Central Dynamo/Dock to Save Station Mines B
+  > Save Station
+      Trivial
+
+> Save Station; Heals? True; Spawn Point
+  > Dock to Central Dynamo
+      Trivial
 
 ----------------
 Metroid Quarantine B
 Asset id: 3141205070
-> Dock to Quarantine Access B; Heals? False
-  * Normal Door to Quarantine Access B/Dock to Metroid Quarantine B
+> Dock to Quarantine Access B; Heals? False; Spawn Point
+  * Plasma Door to Quarantine Access B/Dock to Metroid Quarantine B
+  > Dock to Save Station Mines C
+      Grapple Beam and Space Jump Boots and Morph Ball and Spider Ball and After Metroid Quarantine B Barrier Lowered
+  > Event (Metroid Quarantine B Barrier Lowered)
+      Scan Visor and Grapple Beam and Space Jump Boots and Morph Ball and Spider Ball
 
 > Dock to Elite Quarters Access; Heals? False
-  * Normal Door to Elite Quarters Access/Dock to Metroid Quarantine B
+  * Plasma Door to Elite Quarters Access/Dock to Metroid Quarantine B
+  > Dock to Save Station Mines C
+      Trivial
 
 > Dock to Save Station Mines C; Heals? False
-  * Normal Door to Save Station Mines C/Dock to Metroid Quarantine B
+  * Plasma Door to Save Station Mines C/Dock to Metroid Quarantine B
+  > Dock to Quarantine Access B
+      Space Jump Boots and After Metroid Quarantine B Barrier Lowered
+  > Dock to Elite Quarters Access
+      Trivial
+  > Pickup (Missile)
+      Shoot Super Missile
+
+> Pickup (Missile); Heals? False
+  * Pickup 85; Major Location? False
+  > Dock to Save Station Mines C
+      Trivial
+
+> Event (Metroid Quarantine B Barrier Lowered); Heals? False
+  * Event Metroid Quarantine B Barrier Lowered
+  > Dock to Quarantine Access B
+      Space Jump Boots
 
 ----------------
 Metroid Quarantine A
```

Comment:
For MQA, add helper node called "Front of Spider Track". Connect to nodes as follows:
- Elevator Access B with Spider, Morph, Bombs
- Pickup with Power Bombs > 2, Spider, Space Jump and either (Thermal, Xray or Invisible Objects (Intermediate)
- Quarantine Access A with Space Jump and After Barrier